examples/make_assets/structure_viz.py

```diff
@@ -30,22 +31,19 @@
 
 
 # %% Plot some disordered structures in 2D
-# DEBUG
-mp_api_key = os.environ.get("MP_API_KEY")
-if mp_api_key is None:
-    raise RuntimeError("api key is None")
-
-elif mp_api_key is not None and len(mp_api_key.strip()) == 0:
-    raise RuntimeError("mp api key is empty")
-
-disordered_structs = {
-    mp_id: MPRester().get_structure_by_material_id(mp_id, conventional_unit_cell=True)
-    for mp_id in ["mp-19017", "mp-12712"]
-}
-
-struct: Structure  # for type hinting
-
-for mp_id, struct in disordered_structs.items():
+struct_mp_ids = ("mp-19017", "mp-12712")
+structure_dir = f"{TEST_FILES}/structure"
+
+if not os.environ.get("GITHUB_ACTIONS"):
```

Comment:
`if not os.environ.get("CI"):` would be more generic and compatible with other CI providers. but i think the check shouldn't be for CI at all but rather if the structure files already exist.

Reply:
Brilliant!